### ISSUE TITLE
Tensorboard on Notebook Server

### ISSUE THREAD

**Issue by sachua** [kubeflow/kubeflow#4915]
/kind question

**Question:**
Hi, I managed to deploy Kubeflow on my air gapped, on-prem kubernetes cluster.
When I created a new notebook server and tried to use Tensorboard, I am able to create the Tensorboard instances, but am unable to view them.

From my understanding, the Tensorboard is running inside the Jupyter Notebook docker image, inside a worker pod in my Kubernetes cluster, how will I be able to view my Tensorboard instance?

**Comment by Jeffwan** [comment]
it totally depends on how you "create the Tensorboard instances" if you use tensorboard controller, is a separate pod in your kubernetes cluster. You can visit it by using the istio virtual service path

**Comment by sachua** [comment]
@Jeffwan i used `%load_ext tensorboard` to load the tensorboard notebook extension in my jupyter notebook. I did check the kubeflow tensorboard controller pod but it did not detect the tensorboard instance I had created in my notebook.

How would I go about exposing the port from the Jupyter Notebook image created from the kubeflow UI to the kubeflow tensorboard controller pod? Or is it currently only possible to use tensorboard with kubeflow pipelines?
